flocker/node/test/test_script.py

```diff
@@ -416,7 +416,7 @@ def test_main_starts_service(self):
         """
         service = Service()
         options = ZFSAgentOptions()
-        options.parseOptions([b"example.com"])
+        options.parseOptions([b"1.2.3.4", b"example.com"])
```

Comment:
:question: Positional arguments are so lame, though.  `--local-name ... --control-service ...` is so much more readable.